.gitattributes

```diff
@@ -0,0 +1 @@
+* -text
```

Comment:
Can you provide more context for what this does? 

I looked in [this doc](https://git-scm.com/docs/gitattributes) and saw that the syntax is `pattern attr1 attr2 ...`. What does the `-text` indicate, and what would you like to accomplish?